File: CartScraper/cartridgemodel.py
class Cartridge:
  def __init__(self):
    self.id = None
    self.color = None
    self.formfactor = None
    self.embossedtext = None
    self.frontlabelentry = None
    self.sealofquality = None
    self.mfgstrpresent = None
    self.backlabelentry = None
    self.twodigitcode = None
    self.revision = None
    self.pcb = None
    self.cictype = None
    self.hardware = None
    self.wram = None
    self.vram = None
    self.images = []

  def to_dict(self):
    return {"id": self.id,
            "color": self.color,
            "formfactor": self.formfactor,
            "embossedtext": self.embossedtext,
            "frontlabelentry": self.frontlabelentry,
            "sealofquality": self.sealofquality,
            "mfgstrpresent": self.mfgstrpresent,
            "backlabelentry": self.backlabelentry,
            "twodigitcode": self.twodigitcode,
            "revision": self.revision,
            "pcb": self.pcb,
            "cictype": self.cictype,
            "hardware": self.hardware,
            "wram": self.wram,
            "vram": self.vram,
            "images": self.images}

File: CartDB.Parser/Models/CartridgeModel.cs
using System.Collections.Generic;

namespace CartDB.Parser.Models
{
    public class CartridgeModel
    {
        public int Id { get; set; }
        public string Color { get; set; }
        public string FormFactor { get; set; }
        public string EmbossedText { get; set; }
        public string FrontLabelEntry { get; set; }
        public string SealOfQuality { get; set; }
        public string MfgStrPresent { get; set; }
        public string BackLabelEntry { get; set; }
        public string TwoDigitCode { get; set; }
        public string Revision { get; set; }
        public string Pcb { get; set; }
        public string CICType { get; set; }
        public string Hardware { get; set; }
        public string WRAM { get; set; }
        public string VRAM { get; set; }
        public List<string> Images { get; set; }
        public DeveloperModel Developer { get; set; }
        public PublisherModel Publisher { get; set; }
        public RegionModel Region { get; set; }
        public ProducerModel Producer { get; set; }
        public GameModel Game { get; set; }
        public List<CartridgeChipModel> CartridgeChips { get; set; }
    }
}
